### PR TITLE
Update apps-inc-exl-assignments.md

### PR DESCRIPTION
**Pull request by cekunze**
edited line 51 to read better

### CODE REVIEW

**Review by Erikre — APPROVED** [review]

**Comment by Erikre** [comment]
Thanks for your contribution! #sign-off

File: memdocs/intune/apps/apps-inc-exl-assignments.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Important scenarios exist when including or excluding app assignments:
     For example, if you assign a device group to the **All corporate users user** group, but exclude members in the **Senior Management Staff** user group, **All corporate users** except the **Senior Management staff** get the assignment, because both groups are user groups.
 - Intune doesn't evaluate user-to-device group relationships. If you assign apps to mixed groups, the results may not be what you want or expect.
 
-    For example, if you assign a device group to the **All Users** user group, but exclude an **All personal devices** device group. In this mixed group app assignment, **All users** get the app. The exclusion does not apply.
+For example, if you assign a device group to the **All Users** user group, but exclude an **All personal devices** device group, **All users** get the app. The exclusion does not apply.
 
 As a result, it's not recommended to assign apps to mixed groups.
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/apps-inc-exl-assignments.md:53]:
```suggestion
    For example, if you assign a device group to the **All Users** user group, but exclude an **All personal devices** device group, **All users** get the app. The exclusion does not apply. As a result, it's not recommended to assign apps to mixed groups.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @cekunze, kindly commit the suggested change. Many thanks.

**Comment by yogkumgit** [comment]
@cekunze Please update the document with suggested changes. Thanks

**Review by Erikre — APPROVED** [review]

**Pull request merged**